src/translation-service.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module utils/translation-service
+ */
+
+const translations = {};
+
+/**
+ * Merges package translations to existing ones.
```

Comment:
It doesn't merge anything – it simply adds translations. Do not overcomplicate docs ;).